crates/aptos-faucet/src/lib.rs

```diff
@@ -32,55 +31,57 @@ use aptos_sdk::{
         LocalAccount,
     },
 };
+use clap::Parser;
 use futures::lock::Mutex;
 use reqwest::StatusCode;
-use std::{convert::Infallible, fmt, path::Path, sync::Arc};
-use structopt::StructOpt;
+use std::{convert::Infallible, fmt, path::PathBuf, sync::Arc};
 use url::Url;
 use warp::{http, Filter, Rejection, Reply};
 
 pub mod mint;
 
-#[derive(Debug, StructOpt)]
-#[structopt(
-    name = "Aptos Faucet",
-    author = "Aptos",
-    about = "Aptos Testnet utility service for creating test accounts and minting test coins"
-)]
+/// Aptos Testnet utility service for creating test accounts and minting test coins
+#[derive(Clone, Debug, Parser)]
+#[clap(name = "Aptos Node", author, version)]
 pub struct FaucetArgs {
     /// Faucet service listen address
-    #[structopt(short = "a", long, default_value = "127.0.0.1")]
+    #[clap(short = 'a', long, default_value = "127.0.0.1")]
     pub address: String,
     /// Faucet service listen port
-    #[structopt(short = "p", long, default_value = "80")]
+    #[clap(short = 'p', long, default_value = "80")]
     pub port: u16,
     /// Aptos fullnode/validator server URL
-    #[structopt(short = "s", long, default_value = "https://testnet.aptoslabs.com/")]
-    pub server_url: String,
+    #[clap(short = 's', long, default_value = "https://testnet.aptoslabs.com/")]
+    pub server_url: Url,
     /// Path to the private key for creating test account and minting coins.
     /// To keep Testnet simple, we used one private key for aptos root account
     /// To manually generate a keypair, use generate-key:
     /// `cargo run -p generate-keypair -- -o <output_file_path>`
-    #[structopt(short = "m", long, default_value = "/opt/aptos/etc/mint.key")]
-    pub mint_key_file_path: String,
+    #[clap(
+        short = 'm',
+        long,
+        default_value = "/opt/aptos/etc/mint.key",
```

Comment:
Can we open an issue to overhaul how we handle configs / storage / etc locally? If we're going to have a default path for this stuff, it shouldn't be a path that only works in linux. Again, no need to address this here, but let's open an issue.